src/app/static/src/tests/root/actions.test.ts

```diff
@@ -182,13 +182,31 @@ describe("root actions", () => {
         expect(mockContext.dispatch).not.toHaveBeenCalled();
     });
 
-    it("changes language", async () => {
+    it("changes language and fetch plotting metadata", async () => {
```

Comment:
I'd be inclined to keep checks for `ChangeLanguage` and `getPlottingMetadata` mutations separate, but not overly concerned.

Reply:
Very subjective, but in my mind there are three things to verify:
- Language mutation is fired if language is changed
- Plotting metadata is re-fetched if language changed and data has been loaded
- Plotting metadata is not re-fetched if language changed and not data has been loaded
- 
But this is really subjective, and no problem with your approach if everyone else is happy!

Reply:
We haven't historically been very prescriptive about each test only testing one thing, and often test multiple aspects of a feature in a single test. I think single-check tests are usually cleaner and more readable, but can proliferate, and it's often easier to do multiple checks together particularly if test set up is verbose (though obviously that could usually be factored out into a common method). 

I think multiple-check tests are generally ok, if not ideal, as long as the test title makes clear all the things which are going to be tested.